<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Products Panel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mesressources.qrc">
    <normaloff>:/Images/ressources/index.png</normaloff>:/Images/ressources/index.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QTableWidget" name="tableWidgetProduit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>651</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSupprimerProduit">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>410</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset resource="mesressources.qrc">
      <normaloff>:/Images/ressources/edit-delete.png</normaloff>:/Images/ressources/edit-delete.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonModifierProduit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="icon">
     <iconset resource="mesressources.qrc">
      <normaloff>:/Images/ressources/edit-select-all.png</normaloff>:/Images/ressources/edit-select-all.png</iconset>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageAjout">
     <widget class="QPushButton" name="pushButtonValiderProduit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>83</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset resource="mesressources.qrc">
        <normaloff>:/Images/ressources/list-add.png</normaloff>:/Images/ressources/list-add.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageModif">
     <widget class="QPushButton" name="pushButtonAnnulerProduitModif">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>83</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValiderProduitModif">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>83</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="labelDescriptionProduit">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRayonProduit">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLabel" name="labelUniteMesureProduit">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEditLibelleProduit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPrixUnitaireProduit">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QComboBox" name="comboBoxUniteMesureProduit"/>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QTextEdit" name="textEditDescriptionProduit"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QComboBox" name="comboBoxRayon"/>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButtonAjoutRayon">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelLibelleProduit">
       <property name="text">
        <string>Products label</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushButtonImageProduit">
       <property name="text">
        <string>Browse picture</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QLabel" name="labelImageProduit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>No picture</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPrixUnitaire"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelErrorMessage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion">
    <property name="title">
     <string>Gestion des produits</string>
    </property>
   </widget>
   <addaction name="menuGestion"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mesressources.qrc"/>
 </resources>
 <connections/>
</ui>
